documentation/contributing/introduction.adoc

```diff
@@ -9,13 +9,52 @@ This guide is meant to assist contributors to the Strimzi documentation.
 
 == Project layout
 
+Strimzi uses git to manage repositories.
+
+https://github.com/strimzi/strimzi-kafka-operator[`strimzi-kafka-operator`^] (GitHub):: The public GitHub repo hosts all of the Strimzi code and documentation.
+https://github.com/strimzi/strimzi-kafka-operator/tree/master/documentation[`strimzi-kafka-operator/documentation`^] (GitHub):: The _documentation_ folder is split into _category_ folders to manage the content.
+
+Category folders contain files related to Strimzi guides (Deploying, Quickstart, Overview, Using), and the files that provide the content for one or more of these guides – _assemblies_ and _modules_.
+Assemblies, which usually encapsulate a feature or process, bring the related content contained in modules together.
+
 [source,options="nowrap",subs="+quotes"]
 ----
-quickstart/                          # Quickstart guide
+deploying/                           # Deploying and Upgrading guide
+quickstart/                          # Quick Start guide
+overview/                            # Overview guide
 using/                               # User guide
-assemblies/                          # Assemblies for all guides
-modules/                             # Modules for all guides
+assemblies/                          # Assemblies provide content for all guides
+modules/                             # Modules provide the content for assemblies
 shared/                              # Shared include files
 shared/attributes.adoc               # Global book attributes
 shared/images/                       # Shared image files
 ----
+
+=== Strimzi Overview guide
+
+The intention of the _Strimzi Overview guide_ is for developing an understanding of Strimzi and Apache Kafka.
+The guide does not contain any instructions.
+It provides an overview of the concepts behind Apache Kafka, the Kafka component architecture, and how Strimzi supports Kafka in a Kubernetes cluster.
+The guide also describes how Strimzi _Operators_ help manage a deployment.
+
+The guide contains high-level outlines of the processes required to deploy, configure, secure and monitor a deployment of Strimzi.
+
+=== Deploying Strimzi
```

Comment:
Inconsistency in the title.

Reply:
That's the name at the moment. I've reflected it in the list too.